src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable no-console */
+import classNames from 'classnames';
+import React from 'react';
+import { removeTodo } from '../../api/todos';
+import { Errors } from '../../types/Errors';
+import { Filter } from '../../types/Filter';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[]
+  onSetFilterGlobal: React.Dispatch<React.SetStateAction<string>>
+  selectedFilter: string
+  onSetIsError: React.Dispatch<React.SetStateAction<boolean>>
+  onSetTypeError: React.Dispatch<React.SetStateAction<string>>
+  toLoad:() => Promise<void>
+  onSetisDeletedComplete: React.Dispatch<React.SetStateAction<boolean>>
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  onSetFilterGlobal,
+  selectedFilter,
+  onSetIsError,
+  onSetTypeError,
+  toLoad,
+  onSetisDeletedComplete,
+}) => {
+  const amountActiveTodos = todos.filter(
+    todo => todo.completed === false,
+  ).length;
+  const handleAnchorClick = (
+    e: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+    title: string,
+  ) => {
+    e.preventDefault();
+    onSetFilterGlobal(title);
+  };
+
+  const compTodos = todos.filter(todo => todo.completed);
+
+  console.log(compTodos);
+  const clearCompletedTodos = () => {
+    compTodos.forEach(async (a) => {
+      console.log(a);
+      try {
+        onSetisDeletedComplete(true);
+        await removeTodo(a.id);
+      } catch (inError) {
+        console.log('ERROR DELETE', inError);
+        onSetIsError(false);
+        onSetTypeError(Errors.ErrDEL);
+      }
+
+      onSetisDeletedComplete(false);
+      onSetFilterGlobal(Filter.ALL);
+      toLoad();
```

Comment:
Better to move in your try construction, after removeTodo, because if you don't edlete completed todos, you don't need to reload data from server

Reply:
it's good mark! but not in try cause await is async and flow might go through await and any way i will. maybe in catch?? i'll try